src/test/java/com/redhat/red/build/finder/DistributionAnalyzerTest.java

```diff
@@ -112,4 +112,15 @@ public void loadNestedWAR() throws IOException {
         assertTrue(result == 7);
     }
 
+    @Test
+    public void loadManPageZIP() throws IOException {
+        List<File> target = Collections.singletonList(TestUtils.loadFile("wget.zip"));
```

Comment:
Should this be called "bad.zip" or something that doesn't sound like it's testing the "wget" binary instead of something else.